docs/operations/logging.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Tracing in SlateDB
+---
+
+# Tracing in SlateDB
+
+SlateDB uses the [`tracing`](https://github.com/tokio-rs/tracing) library for logging and diagnostics. This provides structured, contextual logging that helps debug and monitor your database operations.
+
+## Getting Started
+
+To enable logging in your SlateDB application, you'll need to add the following dependencies to your `Cargo.toml`:
+
+```toml
+[dependencies]
+tracing = "0.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+env_logger = "0.10"
+```
+
+## Basic Setup
+
+Here's a basic example showing how to initialize tracing with `env_logger` and SlateDB:
+
+```rust
+use slatedb::Database;
+use tracing::{info, warn, error};
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+
+#[tokio::main]
+async fn main() {
+    // Initialize tracing subscriber
+    tracing_subscriber::registry()
+        .with(tracing_subscriber::EnvFilter::try_from_default_env()
+            .unwrap_or_else(|_| "info".into()))
+        .with(tracing_subscriber::fmt::layer())
+        .init();
+
+    // Now SlateDB operations will emit traces
+    info!("Initializing SlateDB");
+    
+    let db = Database::new()
+        .with_path("my_database")
+        .build()
+        .await
+        .expect("Failed to create database");
+
+    // Your database operations will now emit traces
+    let result = db.get("my_key").await;
+    match result {
+        Ok(value) => info!("Retrieved value successfully"),
+        Err(e) => error!("Error retrieving value: {}", e),
+    }
+}
+```
+
+## Environment Variables
```

Comment:
All of this seems unrelated to SlateDB. It's just about the tracing library. Can you remove it? I think the links at the bottom should be sufficient.

Reply:
sounds good, i took a generic tracing example and tried to apply it here. I will have it updated.